VHAR-7542 Päivitä urapaikkausten taulukon rivien sisältöä

jarnovayrynen opened the pull request:
Päivitä paikkaustoteumien taulukon sarakkeet Figman mukaisiksi

jarnovayrynen committed:
Korjaa suunnitellut hinnan lyhenne

jarnovayrynen committed:
VHAR-7542 Päivitä urapaikkausten taulukon rivien sisältöä

jpluomajoki-solita commented on dev-resources/less/lomake.less:
```diff
@@ -25,13 +25,27 @@
   }
+  .lomakeryhman-otsikko-tausta {
+    background-color: @harmaa12;
+    padding-top: 12px;
```
Näissä voisi käyttää @valistys-x - muuttujia.

jarnovayrynen replied:
Joo, tää oli multa puhtaasti lohkon siirto yhdestä filusta toiseen, niin en hokannu näitä muutella.

jpluomajoki-solita commented on src/cljs/harja/tiedot/urakka/yllapitokohteet/paikkaukset/paikkaukset_paikkauskohteet.cljs:
```diff
@@ -806,3 +805,14 @@
+  Tehdään yksi formatointifunktio, joka osaa näyttää näissä kaikissa tilanteissa luvut lukutilassa oikein.
+  Ja lisäksi näytetään viiva - mikäli arvoa ei ole annettu ollenkaan."
+  [v]
+  (if (vector? v)
```
Olisiko cond siistimpi?

jarnovayrynen replied:
Sama tässä, toisen tekemää koodia vaan siirsin eri paikkaan. Olisihan se siistimpi.

jpluomajoki-solita reviewed (approved):
Vaikuttaa hyvältä, muutama pikkumaininta koodin tyyliin liittyen.

jarnovayrynen committed:
Aseta sama desimaalimäärä massamäärälle paikkaustaulukkoon ja lomakkeeseen

Aseta sama desimaalimäärä (3) massamäärälle paikkaustaulukkoon ja lomakkeeseen
Otsikkokomponenttiin kuitenkin vain kaksi desimaalia

jarnovayrynen committed:
Parantele koodia katselmointikommenttien perusteella

if -> cond
valistys-muuttujat käyttöön LESS:ssä
